Tests/Hotels/Sprint1.feature

```diff
@@ -0,0 +1,84 @@
+@Test @Sprint1
+Feature: Sprint 1 Testcases
+
+@Test18
+Scenario: Verify user can update number of guests on Home page
+    Given I am on Hotels landing page
+    When I click on Travelers
+    Then I enter 6 Adults
+    Then I enter 3 Children
+    Then I select first child age: 4
+    Then I select second child age: Under 1
+    Then I select third child age: 7
+    Then I click Done
+    Then I verify total number of guests in sum of adults and children as same as selected on step #4 and #5
+
+@Test20
+Scenario: Verify TermsAndConditions link and PrivacyStatements link open correct page on new tab
+    Given I am on Hotels landing page
+    When I click on Sign in button
+    Then I click Sign up link
+    Then I click “Terms and Conditions” link
+    Then I verify “Terms and Conditions” page opens in new tab
+    When I click “Privacy Statement” link
+    Then I verify “Privacy Statement” page opens in new tab
+  
+
+
+@Test22
+Scenario: Verify error message for invalid data in SignUp form
+    Given I am on Hotels landing page
+    When I click on Sign in button
+    Then I click Sign up link
+    Then I enter invalid email address
+    Then I enter invalid first name
+    Then I enter invalid last name
+    Then I enter password
+    Then I verify email error is displayed
+    Then I verify first name error is displayed
+    Then I verify last name error is displayed
+    Then I verify “Keep me signed in” checkbox is displayed and enabled
+    Then I verify “Continue” button is displayed but NOT enabled
+
```

Comment:
Have a look here, how I'm labelling steps for above scenario. Pls correct for other scenarios accordingly
@Test22
Scenario: Verify error message for invalid data in SignUp form
    Given I am on Hotels landing page
    When I click on Sign in button
    And I click Sign up link
    And I enter invalid email address
    And I enter invalid first name
    And I enter invalid last name
    And I enter password
    Then I verify email error is displayed
    And I verify first name error is displayed
    And I verify last name error is displayed
    And I verify “Keep me signed in” checkbox is displayed and enabled
    And I verify “Continue” button is displayed but NOT enabled
Again
Precondition for test => Given
Action step => When
Verification step => Then
Any repeated Given/When/Then should be replaced by And.